EDAM_dev.owl

```diff
@@ -53380,6 +53380,8 @@ ows re-sequencing of complete genomes of any given organism with high resolution
 
     <owl:Class rdf:about="http://edamontology.org/topic_0199">
         <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_3321"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/topic_0622"/>
+       
```

Comment:
Would you mind removing this one empty line (purely for consistency of formatting)? Then I'll merge it, looking forward! 😊

Reply:
Ok sure